python/mxnet/foo/nn/conv_layers.py

```diff
@@ -158,6 +158,22 @@ class Conv1D(_Conv):
         see Initializer.
     bias_initializer: Initializer for the bias vector
         see Initializer.
+
+
+    Input Shape:
+        This depends on the `layout` parameter. Input is 3D array of shape
+        (batch_size, channel, width) if `layout` is `NCW`.
```

Comment:
For conv1d,2d,3d operators, I made changes to differentiate input/output channels like in_channel(in_filters) and out_channel(filters). 
For pooling operators, can I just say channel(filters)?